### PR TITLE
sbt: use standard wrapper

### PR DESCRIPTION
**Pull request by catap**
#### Description

This pull requests ride off from macport's sbt wrapper and start use sbt's wrapper.

The goal is simple — introduce support of modern JDK, for example 16+ which doesn't support `-XX:+CMSClassUnloadingEnabled`.

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 11.5.2 20G95 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/ticket/62973) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by catap** [comment]
@reneeotten can I ask you to review this one?

**Review by reneeotten — COMMENTED** [review]
@catap I have no idea what this port does and what issue this PR is trying to solve... so I am likely not the right person to ask ;) There seem to be two different things happening: (1) you grab the sources now from GitHub and make a bunch of changes because of that (none of this is btw listed in the commit message), and (2) some other change that is not clear to me what it's doing - you remove a wrapper script and replace it by what exactly?

**Comment by catap** [comment]
@reneeotten I have no idea who to ask to be honest, because this port was forgotten for a while and I a man who upgraded it last winter ;)

Let me explain that this PR does:
 1. sbt uses github to keep its build artifact. Maven contains only so called launcher which is simple JVM application to run downloader to run sbt.
 2. I migrated from MacPorts wrapper to a standard wrapper which is supported by sbt team.

Main goal is to sync incompatibility of the port with modern java version.

File: devel/sbt/Portfile [hunk 1]
```diff
@@ -56,12 +39,21 @@ destroot {
...
+        file attributes $f -permissions 0755
+    }
 
-livecheck.type  regex
-livecheck.url   https://repo1.maven.org/maven2/org/scala-sbt/sbt-launch/maven-metadata.xml
-livecheck.regex {\<latest\>([0-9.]+)\</latest\>}
+    xinstall -m 0644 ${worksrcpath}/LICENSE ${sbtdir}/LICENSE
+    xinstall -m 0644 ${worksrcpath}/NOTICE ${sbtdir}/NOTICE
+
+    ln -s ${prefix}/share/${name}/bin/${name} ${destroot}${prefix}/bin
```
[the remark this answers is not in the document]
Comment (catap) [reply devel/sbt/Portfile:58]: `${prefix}/share/sbt` contains unpacked artifacts from a team and it contains `bin` folder with the real wrapper.

For example the similar things happened inside [maven](https://github.com/macports/macports-ports/blob/master/java/maven3/Portfile) and [gradle](https://github.com/macports/macports-ports/blob/master/devel/gradle/Portfile).

**Comment by catap** [comment]
Let me be blank. I need this migration to enjoy `SBT_OPTS` environment variable from default sbt wrapper.

I have two options: introudce it inside macport's wrapper or ride of of them and enjoy easy to maintane port.

I prefer the last option :)

**Comment by catap** [comment]
@reneeotten it's added.

**Pull request merged**